organization/src/main/proto/com/improving/app/organization/domain/organizationTypeDefinitions.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
```

Comment:
Here I was trying to match the name used in Tenant service for consistency, but I agree the name could be better. I think `serviceDomainTyes` or something similar would work well

Reply:
The break-out of the protobuf I did made sense to me. I did it after I worked on Tenant and found the message definitions to be unorganized and harder than I would like to find what I needed. If you want to make a change here, that's fine, but I was consciously making a break with the names of Tenant, trying to find a naming scheme that would provide a clear division of responsibility to make it easy to find things in the protobuf definitions and allow for more in the way of a preamble of sealed traits without things getting too messy.

Reply:
I think that was a good move, and it would be good to mirror it in the other services. I just thought that `organizationData` was even. more broad and opaque than `organizationTypeDefinitions` or better yet something like `organizationDomainObjects`. The other two files represent messages rather than objects which represent domain. `Objects` may still be too opaque, we could drop it entirely or replace

Reply:
organizationDomainObjects sounds good